scripts/InitDatabase.sql

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS Jobs;
```

Comment:
We will have to consider change management for the database. In the Alpha a tool called Flyway was used for this.